Terminal.Gui/Views/TextView.cs

```diff
@@ -30,61 +94,81 @@ public bool CloseFile ()
 				throw new ArgumentNullException (nameof (FilePath));
 
 			FilePath = null;
-			_lines = new List<List<Rune>> ();
+			_lines = new List<List<RuneCell>> ();
 			return true;
 		}
 
-		// Turns the string into runes, this does not split the 
+		// Turns the string into cells, this does not split the 
 		// contents on a newline if it is present.
-		internal static List<Rune> ToRunes (string str)
+		internal static List<RuneCell> StringToRuneCells (string str, ColorScheme? colorScheme = null)
 		{
-			List<Rune> runes = new List<Rune> ();
-			foreach (var x in str.ToRunes ()) {
-				runes.Add (x);
+			List<RuneCell> cells = new List<RuneCell> ();
+			foreach (var rune in str.ToRunes ()) {
+				cells.Add (new RuneCell { Rune = rune, ColorScheme = colorScheme });
 			}
-			return runes;
+			return cells;
 		}
 
-		// Splits a string into a List that contains a List<Rune> for each line
-		public static List<List<Rune>> StringToRunes (string content)
+		internal static List<RuneCell> ToRuneCells (IEnumerable<Rune> runes, ColorScheme? colorScheme = null)
 		{
-			var lines = new List<List<Rune>> ();
+			List<RuneCell> cells = new List<RuneCell> ();
+			foreach (var rune in runes) {
+				cells.Add (new RuneCell { Rune = rune, ColorScheme = colorScheme });
+			}
+			return cells;
+		}
+
+		public static List<List<RuneCell>> ToRuneCells (List<RuneCell> cells)
+		{
+			return SplitNewLines (cells);
+		}
+
+		// Splits a string into a List that contains a List<RuneCell> for each line
+		public static List<List<RuneCell>> StringToRuneCellsSplitted (string content, ColorScheme? colorScheme = null)
```

Comment:
This is only called from within TextView, right? I don't think it needs to be `public`.

The word "Splitted" is not an English word. I think this method should be named "StringToLinesOfRuneCells".

Reply:
> This is only called from within TextView, right? I don't think it needs to be public.

Yes but is used by two class, `TextModel` and `TextView`. As other properties and methods if they are used from more than one class is public. Remember that `TextModel` is a private class.

> The word "Splitted" is not an English word. I think this method should be named "StringToLinesOfRuneCells".

Done via https://github.com/gui-cs/Terminal.Gui/pull/2682/commits/51dcec7853cc3d2ee70d6908519ee28bc1afe827.